src/c_api/c_api.cc

```diff
@@ -442,14 +442,14 @@ void PrefixSum(size_t *x, size_t N) {
   delete[] suma;
 }
 
-XGB_DLL int XGDMatrixCreateFromMat_omp(const bst_float* data,  // NOLINT
+XGB_DLL int XGDMatrixCreateFromMat_omp(const bst_float* _data,  // NOLINT
```

Comment:
Why are we renaming `data` to `_data`?

Reply:
The first the I read the code got a little bit confused by namespace data, param data and mat.page_.data.  I will try to fix all leaks from tests in this PR and will change these thing back after done.